libraries/AP_RangeFinder/AP_RangeFinder.h

```diff
@@ -21,9 +21,17 @@
 #include <AP_MSP/msp.h>
 #include "AP_RangeFinder_Params.h"
 
+#ifndef AP_RANGEFINDER_ENABLE
+#define AP_RANGEFINDER_ENABLE 1
+#endif
+
 // Maximum number of range finder instances available on this platform
-#ifndef RANGEFINDER_MAX_INSTANCES
-#define RANGEFINDER_MAX_INSTANCES 10
+#ifndef RANGEFINDER_MAX_INSTANCES 
+  #if AP_RANGEFINDER_ENABLE
+  #define RANGEFINDER_MAX_INSTANCES 10
+  #else
+  #define RANGEFINDER_MAX_INSTANCES 1
+  #endif
```

Comment:
Yeah this will actually work sorry about that on my suggestion!

I usually add a space here to provide a bit of indentation. It seems like that is the convention but I'm not sure.

```suggestion
#ifndef RANGEFINDER_MAX_INSTANCES 
  #if AP_RANGEFINDER_ENABLE
   #define RANGEFINDER_MAX_INSTANCES 10
  #else
   #define RANGEFINDER_MAX_INSTANCES 1
  #endif
```

The reason changing the max. instances saves some space is that we are no longer creating as big parameter storage arrays.

It saved for me around 400B on matek-wing and 1400B on durandal.